<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonBottomGroup</class>
 <widget class="QWidget" name="ButtonBottomGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #437AFF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #1C356F;
}

</string>
   </property>
   <property name="text">
    <string>btn_0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #19BC90;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #0F5A46;
}
</string>
   </property>
   <property name="text">
    <string>btn_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #1F9EEF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #0F3D5C;
}
</string>
   </property>
   <property name="text">
    <string>btn_2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #FF6637;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #9E462C;
}
</string>
   </property>
   <property name="text">
    <string>btn_3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #9B60FB;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #472B70;
}
</string>
   </property>
   <property name="text">
    <string>btn_4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #22A4B3;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #18585F;
}
</string>
   </property>
   <property name="text">
    <string>btn_5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #FF4F4F;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #7F2D2D;
}
</string>
   </property>
   <property name="text">
    <string>btn_6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>12</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #49d0d5;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #386E70;
}
</string>
   </property>
   <property name="text">
    <string>btn_7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>180</width>
     <height>62</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #cbc4ff;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;
}

QPushButton:pressed {
background: #386E70;
}
</string>
   </property>
   <property name="text">
    <string>btn_8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
